Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.py

```diff
@@ -0,0 +1,202 @@
+import json
+import requests
+import urllib3
+import traceback
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from typing import Any, Dict
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+""" CONSTANTS """
+VERSION = "1.0.0"
+
+
+""" CLIENT CLASS """
+
+
+class Client(BaseClient):
+
```

Comment:
Call authentication during client init. To make sure proxy/insecure are used for request of authentication
```suggestion

    def __init__(self, base_url: str, verify: bool, proxy: bool, authentication_url: str, client_id: str, api_key: str):
        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
        payload = {
            "apiKey": api_key,
            "clientID": client_id,
        }
        headers = {"Accept": "application/json", "Content-Type": "application/json"}
        authentication_response = self.get_authentication_token(authentication_url, headers, payload)
        access_token = authentication_response["access_token"]
        headers["Authorization"] = "Bearer " + access_token
        self.headers = headers

```